testutil/validatormock/propose.go

```diff
@@ -165,8 +171,14 @@ func ProposeBlindedBlock(ctx context.Context, eth2Cl eth2wrap.Client, signFunc S
 		return err
 	}
 
-	var pubkey eth2p0.BLSPubKey
-	var block *eth2api.VersionedBlindedBeaconBlock
+	if len(duties) == 0 {
+		return nil
+	}
+
+	var (
+		pubkey eth2p0.BLSPubKey
+		block  *eth2api.VersionedBlindedBeaconBlock
+	)
 	for _, duty := range duties {
```

Comment:
Can we refactor this for loop in order to have it equal to what's happening in `ProposeBlock`?

That is, look up slot into duties, and if not found exit early.

Reply:
Done!